core/src/main/scala/cats/data/NonEmptyList.scala

```diff
@@ -0,0 +1,159 @@
+package cats
+package data
+
+import cats.std.list._
+//it needs a Functor
+//it needs semigroup - combine, filter
+
+/**
+ *  A data type which represents a single element (head) and some other
+ * structure (tail).
+ */
+final case class NonEmptyList[A](head: A, tail: List[A] = List[A]()) {
```

Comment:
👍 agreed.
@ceedubs had also pointed it out here  https://github.com/typelevel/cats/pull/1120#discussion_r66786120
to have apply method in instances

```
def apply[A](head: A, tail: A*) = NonEmptyList(head, tail.toList)
```